src/test/java/ScattTest.java

```diff
@@ -75,4 +75,17 @@ public void testGetJSONObject() {
         JSONObject jsonObj = Utils.getWizardJSONObject();
         assertTrue(jsonObj instanceof org.json.JSONObject);
     }
+    /**
+     * Early Method for testing GUI
+     */
+    /*@Test
+    public void testGui() {
```

Comment:
This was an early idea for a test.  @ejcole @whereswaldon @thehunte199 I spoke with Fenwick today about testing the UI.  He says that he would just keep everything as decoupled from the UI as possible so that we can test what we do with different responses that could have come from the UI.  He doesn't advise trying to monkey with the swing components.  We can test them manually and mostly assume they are doing what they are supposed to.